packages/docs/connections/Elasticsearch.yaml

```diff
@@ -88,6 +90,11 @@ _ref:
 
             Request types:
               - ElasticsearchSearch
+              - ElasticsearchIndex
```

Comment:
Can you please sort requests in alphabetic order? And do the same for the order below

Reply:
✅